src/Apps/Artist/ArtistApp.tsx

```diff
@@ -77,6 +98,7 @@ export const ArtistAppFragmentContainer = createFragmentContainer(ArtistApp, {
       id
       ...ArtistHeader_artist
       ...NavigationTabs_artist
+      ...ArtistRecommendations_artist
```

Comment:
This winds up adding a potentially heavy data load to every request, even if its for an unauthenticated user or a user who doesn't have the lab feature. We can actually prevent against that, and only fetch this fragment if the user has the lab feature.

That concept (does this request need/have access to the recommendations part), can be handled higher up, and here can be referred to as a variable: `...ArtistRecommendations_artist @include(if: $includeRecommendations)`.

For purposes of storybooks and ease of this initial development, you can possibly hard-code that to true (or we can have it read from ENV, or something).

Then, when it comes to actually using it in Force w/ it being actually set properly, you can pass it in via context.

Basically like: https://github.com/artsy/force/blob/203673e3ce5c0c936d51ac3c66e19cbf471e8c70/src/desktop/apps/artist/server.tsx#L33-L36 for the server and https://github.com/artsy/force/blob/203673e3ce5c0c936d51ac3c66e19cbf471e8c70/src/desktop/apps/artist/client.tsx#L13-L16 in the client.

Let me know if you have questions or need help setting that up!

Reply:
Realizing something similar was happening for the 'pricing' lab feature on the artwork page, so just made a similar comment there: https://github.com/artsy/reaction/pull/2222/files#r270593368

Reply:
👍 Thank you! I'll work on this today and reach out with questions.